website/assets/js/components/ajax-button.component.js

```diff
@@ -14,7 +14,8 @@ parasails.registerComponent('ajaxButton', {
   //  ╠═╝╠╦╝║ ║╠═╝╚═╗
   //  ╩  ╩╚═╚═╝╩  ╚═╝
   props: [
-    'syncing'
+    'syncing',
+    'spinner', //
```

Comment:
If we're using the new spinner in all places, lets change it altogether rather than adding a new prop and making this customizable (since there's no need to be able to customize it in HTML to look the old way sometimes)